tests/src/Kernel/ContentLanguageSwitcherTest.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_whitelabel\Kernel;
+
+use Drupal\node\Entity\Node;
+use Drupal\KernelTests\KernelTestBase;
+use Symfony\Component\DomCrawler\Crawler;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Test content language switcher rendering.
+ */
+class ContentLanguageSwitcherTest extends KernelTestBase {
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'block',
+    'content_translation',
+    'language',
+    'locale',
+    'node',
+    'oe_bootstrap_theme_helper',
+    'oe_multilingual',
+    'oe_whitelabel_helper',
+    'oe_whitelabel_multilingual',
+    'system',
```

Comment:
can you double check this test is running in drone, like make it fail and see it runs. Asking because like this it was failing in my local, had to add 'oe_corporate_blocks'

Reply:
It did pass on drone, but locally after rebasing it I do need the same dependency.